### PR TITLE
Signup 160453782

### PR DESCRIPTION
**Pull request by lubwamabenja**
This is my sign up page where the user fills in his/her details to create an account
The css files are inside my css branch
please comment
thanks
 @lubwamabenja, @mwinel, @nuwagalb I request your reviews.

### CODE REVIEW

**Comment by mwinel** [comment]
All the code looks ok though not well organised which makes it harder for reviewers to follow through. Also try to include a way to test this code in your PR.

File: html/signup.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+		<form enctype="">
+			<p>Username</p>
+			<input type="text" name="" placeholder="Enter Username">
+			<p>Email</p>
+			<input type="Email" name="" placeholder="Enter Email address">
+			<p>Password</p>
+			<input type="password" name="" placeholder=" Enter password">
+			<input type="submit" name="" value="Signup">
+			<a href="index.html">Login?</a><br>
+
```
Comment (nuwagalb) [inline html/signup.html:20]: this extra space is unnecessary. I would suggest that you remove it

**Review by nuwagalb — COMMENTED** [review]
I cannot see your css code. Have you added styling to the page?

**Comment by lubwamabenja** [comment]
In my css folder

**Pull request merged**